src/main/java/edu/greenblitz/robotName/commands/arm/roller/ReverseRunRoller.java

```diff
@@ -0,0 +1,20 @@
+package edu.greenblitz.robotName.commands.arm.roller;
+
+import edu.greenblitz.robotName.subsystems.arm.Wrist;
+
+public class ReverseRunRoller extends RollerCommand{
+    @Override
+    public void initialize() {
+        roller.rollOut();
+    }
+
+    @Override
+    public boolean isFinished() {
+        return Wrist.getInstance().isObjectInside();
```

Comment:
and im not sure that if hte object is in the wrist is the correct function for checking if there is an object in the robot